src/components/Examples/example-list.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
```

Comment:
General remark - the JS logo is still much bigger than the other logos.

Reply:
the source .svg you mean? i dont think it matters on the actual rendered site.

Reply:
<img width="1723" alt="Screenshot 2022-06-15 at 13 01 37" src="https://user-images.githubusercontent.com/24988314/173812133-9a6c0dde-5094-4eb2-8615-9e29bbdee2b9.png">
<img width="1717" alt="Screenshot 2022-06-15 at 13 01 20" src="https://user-images.githubusercontent.com/24988314/173812140-dc8d4ec2-4401-427c-8069-573207ad9ae7.png">

if we look at how all icons are rendered, the original size doesn't matter, as @vinckr mentioned. in fact, the original size of all svgs is 32px x 32px - what you're noticing is what i would call a visual, brain based bug, where boxes without rounded corners appear larger than ones with a border radius